Updates linter file

kathyoleary opened the pull request:
📗 Ready to Go,  Minor Linter updates!  Thanks

rylnd commented:
@kathyoleary can you please update the existing linter rules from severity `2` to severity `"error"` ?

rylnd commented:
@kathyoleary it looks like the current build errors are due to a syntax error in the eslintrc file.

camwiegert commented on .eslintrc.js:
```diff
@@ -9,21 +9,23 @@ module.exports = {
-      2,
+      "error",
       "always"
     ]
```
Missing a comma here.

rylnd commented:
I guess I was wrong: you can't interchange `2` with `"error"` in the rule severity.

kathyoleary commented:
Actually, that one is in there EXACTLY how they show it on the Eslint page.

kathyoleary commented:
FYI, cannot have "error" on anyone of them, has to be 2

rylnd commented:
LGTM 👍

kathyoleary commented:
In updating the linter file and fixing it, I also added the recommended from:  https://github.com/LMHT/LMHT.github.io/issues/180
And I went through and added no multiple empty lines, and no unnecessary nested blocks.  The spaces in the parentheses is already covered because the linter has the recommended rules already loaded. 

So if Erica agrees, I'd like to close https://github.com/LMHT/LMHT.github.io/issues/180 when we close this one.

ericanaughton commented:
👍
